ReactWindows/ChakraBridge/ChakraHost.cpp

```diff
@@ -164,6 +96,69 @@ void CALLBACK UnloadSourceCallback(JsSourceContext sourceContext)
     delete context;
 }
 
+JsValueRef CALLBACK NativeRequire(JsValueRef callee, bool isConstructCall, JsValueRef *arguments, unsigned short argumentCount, void *callbackState)
+{
+	ChakraHost* host = (ChakraHost*)callbackState;
+	if (argumentCount != 2)
+	{
+		return JS_INVALID_REFERENCE;
+	}
```

Comment:
Correct behavior is to throw exception.

---
In reply to: [103544640](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103544640) [](ancestors = 103544640)